plugin/tls/tls.go

```diff
@@ -34,6 +34,8 @@ func setTLSDefaults(tls *ctls.Config) {
 		ctls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
 	}
 	tls.PreferServerCipherSuites = true
+	// Allow negotiating http/2 by default.
+	tls.NextProtos = []string{"h2"}
```

Comment:
I believe if we do it here, we don't need to do it in dnsserver/server_grpc.go (but it's a bit hairy to follow that path)

Reply:
... same as `dnsserver/server_https.go`.

Reply:
yeah I was thinking about that. but I agree it's more clear to see it inside the server code instead of only hidden in tls package. So I'd suggest leave duplicate code or remove it from tls plugin